src/site/content/en/metrics/tbt/index.md

```diff
@@ -34,7 +37,7 @@ because the browser cannot interrupt a task that's in progress. So in the event
 that a user _does_ interact with the page in the middle of a long task, the
 browser must wait for the task to finish before it can respond.
 
```

Comment:
Should we also incorporate some reasoning re why between FCP and TTI? This can be in a followup. There has been some confusion around the choice of boundaries.

Reply:
Sure, but let's do it in a followup PR so this content can be published ASAP. Can you file an issue with the suggested changes/reasoning?

Reply:
Filed https://github.com/GoogleChrome/web.dev/issues/3290